lib/versioning/npm/index.spec.ts

```diff
@@ -84,6 +84,14 @@ describe('semver.getNewValue()', () => {
       ['^1.2.3', '4.0.0', '^4.0.0'],
       ['^1.2.3', '4.5.6', '^4.0.0'],
       ['^1.0.0', '4.5.6', '^4.0.0'],
+
+      ['^0.2.3', '0.2.4', '^0.2.4'],
+      ['^2.3.0', '2.4.0', '^2.4.0'],
+      ['^2.3.4', '2.4.5', '^2.4.0'],
```

Comment:
These shouldn't really happen with `replace` though, should they?

Reply:
Fixed, now it remains `^2.3.4`